### PR TITLE
Add dodo

### LINKED ISSUE #9: 🐦 Add dodo
The build is a little brittle right now: adding a `doit` approach would:
- help ensure running the same code locally, in ci, and on binder
- clearly define the dependencies between build artifacts

> As a follow-on to #4, #2, and probably a pre-requisite to more robust testing, docs, etc.

### PR DESCRIPTION
**Pull request by nrbgt**
- [Binder](https://mybinder.org/v2/gh/nrbgt/ipyelk/gh-9-add-dodo?urlpath=lab)
- fixes #9
- adds doit with lots of goodies

### CODE REVIEW

**Comment by nrbgt** [comment]
merged `master`... (just a merge commit). this is probably gtg if it passes, and if there is no issue with the premise of #9 in the first place....

File: dodo.py [hunk 1]
```diff
@@ -0,0 +1,402 @@
+""" doit tasks for ipyradiant
```
Comment (dfreeman06) [inline dodo.py:1]: reference to ipyradiant

File: dodo.py [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+
+
+def task_test():
+    """ run all the notebooks
+    """
+
+    def _nb_test(nb):
+        def _test():
+            env = dict(os.environ)
+            env.update(IPYRADIANT_TESTING="true")
```
Comment (dfreeman06) [inline dodo.py:179]: maybe convert to a `JUPYRDF_TESTING`?

Comment (nrbgt) [reply dodo.py]: doh! 0e3e3cb

File: dodo.py (hunk 1 above)
Comment (nrbgt) [reply dodo.py]: 0e3e3cb

**Comment by nrbgt** [comment]
yeah, getting watch working right is a PITA, but glad it's working alright. Sheepishly have not tested on windows, but have with the ipyradiant stuff.

**Pull request merged**